src/components/Products.js

```diff
@@ -47,8 +47,8 @@ const Products = ({products, setProducts}) => {
                                                 ${product.price}
                                             </div>
                                             <div className="productButtonsContainer">
-                                                <NavLink to={`/products/${product.id}`} className="vProdButtonAll">View Product</NavLink>
-                                                <button className="addToCartButton">Add to Cart</button>
+                                                <NavLink to={`/products/${product.id}`} className="button-81">View Product</NavLink>
```

Comment:
Why is this specific button different than the rest?

Reply:
button-81 is a classname provided by the free CSS found. I can re move the 81 to just leave button as the classname to remove confusion. The button will now highlight when hovered with this css.